html/captive-portal/templates/oauth2/landing.html

```diff
@@ -1,4 +1,7 @@
-<h4>[% i18n_format("You are about to authenticate against %s", source.description) %]</h4>
+<!-- [% source.type %] -->
+<div class="layout--center">
+  <h4>[% i18n_format("You are about to authenticate against %s", source.description) %]</h4>
```

Comment:
I guess you can remove this since the title is already put in an h4 inside the layout.

We're getting double the info because of that

![image](https://cloud.githubusercontent.com/assets/3857942/14051244/cd57d0ac-f297-11e5-89b7-bc50bce13bbd.png)